content/en/blog/2023/end-user-discussions-03.md

```diff
@@ -0,0 +1,273 @@
+in the Collector, as it covers a lot of metrics, and has no dependencies. One
+user noticed a 20% reduction in CPU usage when they moved from a vendor-specific
+agent to the host metrics receiver for infrastructure monitoring.
+
+#### 3 - Auto-instrumentation
+
+**Q:** Is there a way to automatically create spans without code changes?
+
+**A:** It depends on the use cases:
+
+- Auto instrumentation options are maturing in OTel; for example, the Java JAR
+- agent takes care of instrumenting
+  [most libraries](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks)
+  that are used by applications.
+- If you’re using Kubernetes, they can use the
+  [OTel operator](https://github.com/open-telemetry/opentelemetry-operator),
+- which takes care of instrumentations for applications deployed on k8s.
+- If you’re using AWS lambda, you should check out the
+  [OTel Lambda extension](https://github.com/open-telemetry/opentelemetry-lambda).
+
+#### 4 - Leveraging telemetry from OTel
+
+**Q:** Has there been work toward tele-command standards to leverage the
+telemetry from OTel?
+
+**A:** Check out
+[this paper](https://www.gsse.biz/pdfs/papers/DASIA2018-abstract.pdf), and
+[OpAMP](https://opentelemetry.io/blog/2022/opamp/).
+
+#### 5 - Message brokers
+
+**Q:** What are some use cases for message brokers?
+
+**A:** IoT use cases (car manufacturer). There is also ongoing work for semantic
+conventions support for messages.
+
+### UPDATES AND COMMUNICATIONS
+
+#### 1 - Unified query standard
+
+**Q:** Is there an update on the upcoming Unified Query Standard working group
+for observability data and discussion at O11y Day at KubeCon EU?
+
+**A:** The Observability TAG within CNCF is working to launch a working group
+that is going to analyze the various query languages that are out there and come
+up with use cases, such as, what are your most common alert and diagnostic
+types, and what are some uncommon patterns that you’d like to have available?
+Then, we’d like to see if there’s any way we can come up with a recommendation
+for a unified standard language across vendors. Maybe SQL-ish?
+
+We’re officially launching the working group at the end of the month; the
+charter is open for comments.
+[View here](https://docs.google.com/document/d/1JRQ4hoLtvWl6NqBu_RN8T7tFaFY5jkzdzsB9H-V370A).
+We are going to start making the conference circuit and gather feedback, the
+first place will be at
+[Observability Day](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/observability-day/).
+Join the discussion at
+[#telemetry-analysis](https://cloud-native.slack.com/archives/C04LXHPDW6M) in
+CNCF’s Slack instance.
+
+#### 2 - Documentation and searches
+
+**Q:** Where do you go to find documentation and answers to your questions?
+
+**A:** It would be helpful to gather feedback from end users – do you go to the
```

Comment:
This reads more like a question than an answer, so I'm finding it a tad confusing. 😵‍💫